webapp/src/tutorial.tsx

```diff
@@ -355,14 +397,20 @@ export class TutorialCard extends data.Component<TutorialCardProps, TutorialCard
         this.closeLightbox();
         let th = this.refs["tutorialhint"] as TutorialHint;
         if (th && th.state && th.state.visible) {
-            this.setState({ showHintTooltip : true });
-            th.elementRef.removeEventListener('click', this.expandedHintOnClick);
-            document.removeEventListener('click', this.hintOnClick);
+            if (th.elementRef) {
+                document.removeEventListener('click', this.hintOnClick);
```

Comment:
we'll need to review this for accessibility. i think Sam had some helper to fire onClick when Enter/space is pressed too.